back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -55,8 +58,11 @@ def update
 
     def update_all
       authorize CustomField.new(resource: @custom_form), :update_all?, policy_class: IdeaCustomFieldPolicy
+      participation_method = Factory.instance.participation_method_for @custom_form.participation_context
+      verify_no_responses participation_method
+
       update_fields
-      @custom_form.reload
+      @custom_form.reload if @custom_form.persisted?
```

Comment:
Why is this condition required?

Reply:
There was a bug with the endpoint. When deleting all custom fields from a form that was not customized yet (and therefore doesn't exist), the endpoint failed with 500.